leaf-ui/rappid-extensions/ConfigInspector.js

```diff
@@ -101,13 +101,17 @@ var Config = Backbone.View.extend({
     initialize: function(){
         this.innerView = new ResultsDropdown({collection:this.model.get("results"), config: this.model});
         this.innerView.render();
+        // removes configurations
```

Comment:
I don't know if these comments are necessary/accurate. I defer to @amgrubb on that though. If you leave them in the correct style is `// First word capitalized`